packages/fluentui/react-northstar/src/themes/teams/components/List/listItemVariables.ts

```diff
@@ -53,9 +53,11 @@ export default (siteVariables: any): ListItemVariables => {
     contentMediaLineHeight: siteVariables.lineHeightSmall,
 
     // Selectable
-    selectableFocusHoverColor: siteVariables.colors.white,
-    selectableFocusHoverBackgroundColor: siteVariables.colors.brand[500],
-    selectedColor: siteVariables.bodyColor,
-    selectedBackgroundColor: siteVariables.colors.grey[100],
+    selectableFocusHoverColor: siteVariables.colorScheme.default.foregroundHover,
+    selectableFocusHoverBackgroundColor: siteVariables.colorScheme.default.backgroundHover,
+
+    // Selected
+    selectedColor: siteVariables.colorScheme.default.foregroundPressed,
```

Comment:
Why not `foregroundActive` or `foregroundActive1` since bellow we are using `backgroundActive1`? 🤔

Reply:
You can think of the 'selected' state as a pressed state. And that's the design :)